src/Cli/Utils.cs

```diff
@@ -898,17 +898,18 @@ public static bool TryMergeConfigsIfAvailable(out string mergedConfigFile)
             {
                 string baseConfigFile = RuntimeConfigPath.DefaultName;
                 string environmentBasedConfigFile = RuntimeConfigPath.GetFileName(environmentValue, considerOverrides: false);
-                mergedConfigFile = RuntimeConfigPath.GetMergedFileNameForEnvironment(CONFIGFILE_NAME, environmentValue);
 
                 if (DoesFileExistInCurrentDirectory(baseConfigFile) && !string.IsNullOrEmpty(environmentBasedConfigFile))
                 {
                     try
                     {
                         string baseConfigJson = File.ReadAllText(baseConfigFile);
                         string overrideConfigJson = File.ReadAllText(environmentBasedConfigFile);
+                        _logger.LogInformation($"Merging {baseConfigFile} and {environmentBasedConfigFile}");
                         string mergedConfigJson = Merge(baseConfigJson, overrideConfigJson);
-
+                        mergedConfigFile = RuntimeConfigPath.GetMergedFileNameForEnvironment(CONFIGFILE_NAME, environmentValue);
                         File.WriteAllText(mergedConfigFile, mergedConfigJson);
+                        _logger.LogInformation($"Generated merged config file: {mergedConfigFile}");
```

Comment:
is it possible to log the full file path? This would be a good place to start with that enhancement, and other locations can follow in future PRs.